src/main/scala/org/camunda/feel/impl/builtin/TemporalBuiltinFunctions.scala

```diff
@@ -98,4 +94,73 @@ class TemporalBuiltinFunctions(clock: FeelEngineClock) {
     ValDate(date.`with`(TemporalAdjusters.lastDayOfMonth()))
   }
 
+  private def timestampFunction = builtinFunction(
+    params = List.empty,
+    invoke = { case _ =>
+      val timestamp = clock.getCurrentTime.toEpochSecond
+      ValNumber(timestamp)
+    }
+  )
+
+  private def timestampFunction2 = builtinFunction(
+    params = List("datetime"),
+    invoke = {
+      case List(ValDateTime(datetime))        =>
+        val timestamp = datetime.toEpochSecond
+        ValNumber(timestamp)
+      case List(ValLocalDateTime(datetime))   =>
+        val timestamp = datetime.toInstant(ZoneOffset.UTC).getEpochSecond
+        ValNumber(timestamp)
+      case _                                  => ValNull
+
+    }
+  )
+
+  private def timestampFunction3 = builtinFunction(
+    params = List("timestamp","zoneId"),
+    invoke = {
+
+      case List(ValString(timestamp),ValString(zoneId))    =>
+        System.out.println("timestamp---"+timestamp+"---"+zoneId)
+        var instant = Instant.ofEpochSecond(timestamp.toLong)
+        if (timestamp.length() == 10) {
+          instant = Instant.ofEpochSecond(timestamp.toLong)
+        } else if (timestamp.length() == 13){
+          instant = Instant.ofEpochMilli(timestamp.toLong)
+        }
+        if(!"UTC".equals(zoneId) && zoneId.nonEmpty) {
+          val localDateTime = ZonedDateTime.ofInstant(instant, ZoneId.of(zoneId))
+          System.out.println(localDateTime)
+          ValDateTime(localDateTime)
+        }else {
+          val localDateTime = instant.atZone(ZoneId.of("UTC")).toLocalDateTime
+          System.out.println(localDateTime)
+          ValLocalDateTime(localDateTime)
+        }
+      case _                                               => ValNull
+
+    }
+  )
+
+  private def timestampMilliFunction = builtinFunction(
+    params = List.empty,
+    invoke = { case _ =>
+      val timestamp =  clock.getCurrentTime.toInstant.toEpochMilli
+      ValString(timestamp.toString)
+    }
+  )
+
+  private def timestampMilliFunction2 = builtinFunction(
+    params = List("datetime"),
+    invoke = {
+      case List(ValDateTime(datetime))        =>
+        val timestamp =  datetime.toInstant.toEpochMilli
+        ValString(timestamp.toString)
+      case List(ValLocalDateTime(datetime))   =>
+        val timestamp =  datetime.toInstant(ZoneOffset.UTC).toEpochMilli
```

Comment:
:question: Why do we need a function to convert the date into milliseconds?

A UNIX timestamp is represented in seconds.